scripts/build_renewable_profiles.py

```diff
@@ -499,7 +499,11 @@ def create_scaling_factor(
     regions = regions.set_index("name").rename_axis("bus")
 
     cutout = atlite.Cutout(paths["cutout"])
-    cutout = filter_cutout_region(cutout, regions)
+    if not snakemake.wildcards.technology.startswith("hydro"):
+        # if technology is not hydro, restrict the region of the cutout
+        # hydrobasins may span beyond the region of the country, so
+        # it is unsafe to restrict the region for hydro
```

Comment:
Could it make sense to re-formulate a bit the comment for better readability? Something like "the region should be restricted for non-hydro technologies, while hydro potential is calculated across hydrobasins which may span beyond the region of the country"?

Reply:
Done